intelmq/bots/collectors/rt/collector_rt.py

```diff
@@ -22,17 +25,35 @@ def init(self):
 
         self.set_request_parameters()
 
+        if getattr(self.parameters, 'search_newer_than', None):
+            try:
+                self.newer_than = parser.parse(self.parameters.search_newer_than)
+                self.newer_than_type = 'absolute'
+            except ValueError:
+                self.newer_than_relative = timedelta(minutes=FilterExpertBot.parse_relative(self.parameters.search_newer_than))
```

Comment:
IMHO we should have an error message for the admin/user if neither the search_newer_than field is relative nor absolute.